src/SingleBookingPage/bookingItem/Header.js

```diff
@@ -11,6 +11,11 @@ type Props = {|
   booking: Header_booking,
 |};
 
+const formatBookingId = (databaseId: number): string => {
```

Comment:
you can move this function to `/helpers` we'll need it in more places

Reply:
Yes, another name would be better, or it's okay so? I also thought to add another parameter to decide the "slices". What do you think?

Thanks @jalbertsr